akka-cluster/src/multi-jvm/scala/akka/cluster/MultiNodeClusterSpec.scala

```diff
@@ -91,8 +92,11 @@ object MultiNodeClusterSpec {
   }
 }
 
-trait MultiNodeClusterSpec extends Suite with STMultiNodeSpec with WatchedByCoroner {
-  self: MultiNodeSpec =>
+abstract class MultiNodeClusterSpec(multiNodeconfig: MultiNodeConfig)
+    extends MultiNodeSpec(multiNodeconfig)
+    with Suite
+    with STMultiNodeSpec
+    with WatchedByCoroner {
```

Comment:
This is the main change. Rest is mechanical adjustments for `extends MultiNodeSpec with MultiNodeClusterSpec`